include/openthread/srp_server.h

```diff
@@ -298,6 +308,27 @@ otError otSrpServerSetAnycastModeSequenceNumber(otInstance *aInstance, uint8_t a
  */
 void otSrpServerSetEnabled(otInstance *aInstance, bool aEnabled);
 
+/**
+ * This function returns SRP server TTL configuration.
+ *
+ * @param[in]   aInstance   A pointer to an OpenThread instance.
+ * @param[out]  aTtlConfig  A pointer to an `otSrpServerTtlConfig` instance.
+ *
+ */
+void otSrpServerGetTtlConfig(otInstance *aInstance, otSrpServerTtlConfig *aTtlConfig);
+
+/**
+ * This function sets SRP server TTL configuration.
```

Comment:
@jwhui it may worth documenting the expected behavior when a TTL from client exceeds the configured range here, like we do for `otSrpServerSetLeaseConfig`. It will also be useful to document that the max TTL larger than the max lease will be ignored when granting the TTL to a client (same for the min TTL).